diff --git cmake/FindPython.cmake cmake/FindPython.cmake
index 152dc0e51..2059aee37 100644
--- cmake/FindPython.cmake
+++ cmake/FindPython.cmake
@@ -45,9 +45,6 @@ endif(Python_FIND_REQUIRED)
 # Search for the Python interpreter.
 find_package(PythonInterp ${_options})
 
-# Search for the Python header files and libraries.
-find_package(PythonLibs ${_options})
-
 # Set PYTHON_INCLUDE_DIRS variable, because FindPythonLibs does not do it.
 set(PYTHON_INCLUDE_DIRS "${PYTHON_INCLUDE_DIR}")
 
diff --git python/CMakeLists.txt python/CMakeLists.txt
index db824fd2e..3ec8155eb 100644
--- python/CMakeLists.txt
+++ python/CMakeLists.txt
@@ -22,22 +22,7 @@ set(Python_FIND_VERSION 2)
 set(PythonInterp_FIND_VERSION_MAJOR 2)
 find_package(Python REQUIRED)
 if (PYTHONINTERP_FOUND)
-    find_package(Boost REQUIRED)
-    if (${Boost_MAJOR_VERSION} STREQUAL 1 AND ${Boost_MINOR_VERSION} STRGREATER 66)
-        # Boost>=1.67 Python components require a Python version suffix
-	set(BOOST_PYTHON_LIBRARY_NAME
-		python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}
-		CACHE
-		FILEPATH
-	       	"The name of the boost python library to search for")
-    else ()
-	set(BOOST_PYTHON_LIBRARY_NAME
-		python
-		CACHE
-		FILEPATH
-		"The name of the boost python library to search for")
-    endif ()
-    find_package(Boost REQUIRED COMPONENTS ${BOOST_PYTHON_LIBRARY_NAME})
+    find_package(Boost REQUIRED COMPONENTS python)
 
     find_package (NUMPY REQUIRED)
 
